config/config.example.toml

```diff
@@ -184,6 +184,15 @@ trustpay.base_url = "https://test-tpgw.trustpay.eu/"
 trustpay.base_url_bank_redirects = "https://aapi.trustpay.eu/"
 zen.base_url = "https://api.zen-test.com/"
 
+[bank_config.online_banking_czech_republic]
```

Comment:
where are these configs going to be used?

Reply:
It will be used by the payment method list api, I missed to add these config while adding new payment methods for Adyen